### PR TITLE
Update the code to remove some deprecation notices

### PR DESCRIPTION
**Pull request by floriansimon1**

### CODE REVIEW

File: src/semafour.cc [hunk 1]
```diff
@@ -14,15 +14,19 @@ using v8::Persistent;
 using v8::String;
 using v8::Uint32;
 using v8::Value;
+using v8::False;
```
Comment (cjihrig) [inline src/semafour.cc]: These aren't used in the code. Can you remove them from this PR please.

File: src/semafour.cc [hunk 2]
```diff
@@ -14,15 +14,19 @@ using v8::Persistent;
...
+using v8::True;
 
 Persistent<Function> Semafour::constructor;
 
+#define TRYWAIT_OK     0
```
Comment (cjihrig) [inline src/semafour.cc]: Same with these.

**Review by cjihrig — COMMENTED** [review]
Left a couple of comments. Can you also rebase this? I did some maintenance recently and dropped support for Node < 8.

**Comment by floriansimon1** [comment]
Sure, I will.

Le dim. 16 déc. 2018 à 10:39, Colin Ihrig <notifications@github.com> a
écrit :

**Comment by cjihrig** [comment]
Landed in https://github.com/cjihrig/semafour/commit/441ffa74c87440eabd09188f9bd2bbb890f39f7c. Thanks!

**Pull request closed without merging**